src/main/java/com/exasol/containers/ssh/IdentityProviderMock.java

```diff
@@ -6,7 +6,7 @@
 /**
  * Enables to add an identity to a java secure channel {@link JSch}.
  */
-public interface IdentityProvider {
+public interface IdentityProviderMock {
```

Comment:
Why rename this to `...Mock`? This is production code in `src/main/java`.

Reply:
Auto-refactoring gone wrong. IntelliJ was a little bit to eager here it seems.